buildtest/utils/command.py

```diff
@@ -1,7 +1,69 @@
+from buildtest.utils.file import read_file
```

Comment:
nit pick: can we have ``from buildtest.utils.file import read_file``  after ``imports``